go.mod

```diff
@@ -123,21 +124,35 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 )
 
+// the sdk's mandatory replaces
 replace (
-	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
-	github.com/aws/aws-sdk-go v1.25.48 => github.com/aws/aws-sdk-go v1.33.0
-	github.com/aws/aws-sdk-go v1.27.0 => github.com/aws/aws-sdk-go v1.33.0
-	github.com/cosmos/cosmos-sdk => github.com/classic-terra/cosmos-sdk v0.45.13-classic
-	github.com/cosmos/ledger-cosmos-go => github.com/terra-money/ledger-terra-go v0.11.2
-	github.com/docker/docker v1.4.2-0.20180625184442-8e610b2b55bf => github.com/docker/docker v1.6.1
-	github.com/ethereum/go-ethereum v1.9.25 => github.com/ethereum/go-ethereum v1.10.9
-	github.com/gin-gonic/gin v1.4.0 => github.com/gin-gonic/gin v1.7.7
+	// use cosmos fork of keyring
+	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	// dgrijalva/jwt-go is deprecated and doesn't receive security updates.
+	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
+	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
+
+	// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
+	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
+	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
+
+	// Use regen gogoproto fork
+	// This for is replaced by cosmos/gogoproto in future versions
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/libp2p/go-buffer-pool v0.0.2 => github.com/libp2p/go-buffer-pool v0.1.0
-	github.com/microcosm-cc/bluemonday v1.0.2 => github.com/microcosm-cc/bluemonday v1.0.16
-	github.com/miekg/dns v1.0.14 => github.com/miekg/dns v1.1.25
-	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
-	github.com/tendermint/tendermint => github.com/classic-terra/tendermint v0.34.24-terra.0
-	github.com/tidwall/gjson v1.6.7 => github.com/tidwall/gjson v1.9.3
+
+	// use a secure protoreflect version
+	github.com/jhump/protoreflect => github.com/jhump/protoreflect v1.9.0
+
+	// use grpc compatible with regen gogoproto fork
 	google.golang.org/grpc => google.golang.org/grpc v1.33.2
 )
+
+// replaces that lunc needs
+replace (
+	// use fork of cosmos-sdk with lunc's changes
+	github.com/cosmos/cosmos-sdk => github.com/classic-terra/cosmos-sdk v0.45.13-classic
+	// use a ledger library that uses coin-type 330
+	github.com/cosmos/ledger-cosmos-go => github.com/terra-money/ledger-terra-go v0.11.2 // TODO: bring this up to date with github.com/cosmos-ledger-cosmos-go
+	// use a version of tendermint that is patched for lunc
+	github.com/tendermint/tendermint => github.com/classic-terra/tendermint v0.34.24-terra.0 // TODO: minimum safe version of tendermint is v0.34.26, see release notes at https://github.com/informalsystems/tendermint
```

Comment:
remember to update this before making a release

Reply:
That has already been sorted @ https://github.com/classic-terra/core/blob/04401e307f1c8c63446ddc8087454363618eb73a/go.mod#L140